bundles/org.openhab.binding.mqtt.homie/src/main/resources/OH-INF/thing/homie-channels.xml

```diff
@@ -7,11 +7,19 @@
 	<channel-type id="homie-boolean">
 		<item-type>Switch</item-type>
 		<label>Boolean</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Power</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="homie-color">
 		<item-type>Color</item-type>
 		<label>Color Value (Red,Green,Blue)</label>
+		<tags>
+			<tag>Control</tag>
```

Comment:
Color is fine for this, but I wouldn't tag it as control (it could be read only)